ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml

```diff
@@ -2032,26 +2052,26 @@ tests:
     test:
     - ref: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-ibmcloud-ipi-ovn-mtu-migrate
-- as: ibmcloud-ipi-private-fips-f14
-  cron: 27 10 13,28 * *
+- as: ibmcloud-ipi-ep-fips-f14
```

Comment:
The "ep" in the job name may be a little unclear for people who is not familiar with the detailed workflow, considering it's still a short job name, how about using "custom-endpoint" directly, or even using "custom-cos-endpoint" as we're only overriding the COS endpoint.

Reply:
updated.